Kickstarter-iOS/AppDelegate.swift

```diff
@@ -388,6 +392,21 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
     task.resume()
   }
 
+  private func updateFirebaseConsent(status: AppTrackingAuthorization) {
+    // https://developers.google.com/tag-platform/security/guides/app-consent?platform=ios&consentmode=advanced#set_up_consent_mode
+    let consentStatus: ConsentStatus = if case .authorized = status {
+      .granted
+    } else {
+      .denied
+    }
```

Comment:
Nice. I appreciate the link to the docs. Also, still getting used to reading this new `if case` syntax!

Reply:
It's functionally the same as:

```swift
let consentStatus: ConsentStatus
if case .authorized = status {
  consentStatus = .granted
} else {
  consentStatus = .denied
}
```

I think it's a little cleaner because it means that the variable is never in that weird in-between uninitialized state, but if we don't like it we can always undo these or make a formatting rule for them.